ca/certificate-authority.go

```diff
@@ -596,7 +596,17 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(csr x509.CertificateRequest
 	}
 
 	// Submit the certificate to any configured CT logs
-	go ca.Publisher.SubmitToCT(certDER)
+	go func() {
```

Comment:
RPC comms errors (timeouts, parse problems) are logged internally in the RPC layer, so we don't need to log them specially.